src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3564,72 +3797,72 @@ def main():
+        msg = ("Job value error in %s handling %s event" %
+               (event.hook_name, hooks.event_name(event.type)))
         if jobutil is not None:
-            msg += (" for job %s" % (e.job.id))
+            msg += (" for job %s" % (event.job.id))
             try:
-                e.job.delete()
-                msg += " (deleted)"
+                event.job.Hold_Types = pbs.hold_types("s")
+                event.job.rerun()
+                msg += " (held)"
             except:
-                msg += " (delete failed)"
+                msg += " (hold failed)"
         msg += (": %s %s" % (exc.__class__.__name__, str(exc.args)))
         pbs.logmsg(pbs.EVENT_ERROR, msg)
-        e.reject(msg)
-    except JobValueError, exc:
-        # Something in PBS is misconfigured
+        event.reject(msg)
+    except UserError, exc:
+        # User must correct problem and resubmit job
         pbs.logmsg(pbs.EVENT_DEBUG,
                    str(traceback.format_exc().strip().splitlines()))
-        msg = ("Job value error in %s handling %s event" %
-               (e.hook_name, hooks.event_name(e.type)))
+        msg = ("User error in %s handling %s event" %
+               (event.hook_name, hooks.event_name(event.type)))
         if jobutil is not None:
-            msg += (" for job %s" % (e.job.id))
+            msg += (" for job %s" % (event.job.id))
             try:
-                e.job.Hold_Types = pbs.hold_types("s")
-                e.job.rerun()
-                msg += " (held)"
+                event.job.delete()
+                msg += " (deleted)"
             except:
-                msg += " (hold failed)"
+                msg += " (delete failed)"
         msg += (": %s %s" % (exc.__class__.__name__, str(exc.args)))
         pbs.logmsg(pbs.EVENT_ERROR, msg)
-        e.reject(msg)
+        event.reject(msg)
     except Exception, exc:
         # Catch all other exceptions and report them.
         pbs.logmsg(pbs.EVENT_DEBUG,
                    str(traceback.format_exc().strip().splitlines()))
         msg = ("Unexpected error in %s handling %s event" %
-               (e.hook_name, hooks.event_name(e.type)))
+               (event.hook_name, hooks.event_name(event.type)))
         if jobutil is not None:
-            msg += (" for job %s" % (e.job.id))
+            msg += (" for job %s" % (event.job.id))
             try:
-                e.job.Hold_Types = pbs.hold_types("s")
-                e.job.rerun()
+                event.job.Hold_Types = pbs.hold_types("s")
+                event.job.rerun()
                 msg += " (held)"
             except:
                 msg += " (hold failed)"
         msg += (": %s %s" % (exc.__class__.__name__, str(exc.args)))
         pbs.logmsg(pbs.EVENT_ERROR, msg)
-        e.reject(msg)
+        event.reject(msg)
 
 # The following block is skipped if this is a unit testing environment.
 if __name__ == "__builtin__":
-    start_time = time.time()
+    START = time.time()
```

Comment:
I wouldn't use all caps here.  The other place you use all caps is for global variables.  This is a local

Reply:
Because it doesn't appear in a function or method, pylint assumes it is a static and wants it in all caps. I can leave it in caps to keep pylint quiet or I can change it to lower case. Which would you prefer?